test/result/result_test.dart

```diff
@@ -263,7 +263,7 @@ void main() {
   test('handle unary', () {
     var result = ErrorResult('error', stack);
     var called = false;
-    result.handle((error) {
+    result.handle((Object? error) {
```

Comment:
Does this work if the argument is non-nullable?

`error` instances should never be `null`, and the `stackTrace` should always be a `StackTrace`, but I don't recall if some of the APIs require more loosely typed callbacks

Reply:
There were a BUNCH of weird test failures here. It was really annoying.

A agree. 🤷